drip/drips.py

```diff
@@ -229,6 +230,17 @@ def _get_unsubscribe_link_campaign(self) -> Optional[str]:
         unsubscribe_link = validate_path_existence("unsubscribe_campaign", url_args)
         return unsubscribe_link
 
+    def _get_unsubscribe_link(self) -> Optional[str]:
+        """
+        Generate url for Unsubcribe to app with user data and validates existence of this url in project.
+        Checking if it was configured by the user.
+        """
+        email_token = EmailToken(self.user)
+        uidb64, token = email_token.get_uidb64_token_user_only()
+        url_args = {"uidb64": uidb64, "token": token}
+        unsubscribe_link = validate_path_existence("unsubscribe_app", url_args)
+        return unsubscribe_link
```

Comment:
Isn't this code repeated in other places? I ask because I don't remember. If it is maybe it would be nice to avoid this repetition

Reply:
It is not the same code repeated is similar but not the same.